industry_benchmarks/utils/fix_networks.py

```diff
@@ -144,13 +145,19 @@ def _check_and_deduplicate_transforms(
     for t_name, t_list in transforms_dict.items():
         if len(t_list) != 3:
             # TODO: we can turn this into a warning message if it's too painful
-            errmsg = (
-                f"Too few transformations found for {t_name} "
-                "this indicates a partially completed set of results. "
-                "Please ensure that your input network is finished and "
-                "any reproducible partial failures have been removed."
-            )
-            raise ValueError(errmsg)
+            if allow_missing:
+                errmsg = (f"Too few transformations found for {t_name}"
+                          f"this indicates a partially complete set of results."
+                          f"This edge will be replaced.")
```

Comment:
I think the edge may not necessarily be replaced, but only if removing the edge would lead to disconnected networks. Maybe instead something like the edge will be ignored/treated as if it none of the simulations of this edge completed successfully?